spec/router_spec.js

```diff
@@ -321,6 +321,74 @@ describe('Router', function() {
         router.params({a: undefined, b: '22', c: false, d: null, e: 0});
         expect(router.params()).toEqual({b: '22', e: 0});
       });
+
+
+      describe('with replaceState', function() {
```

Comment:
Actually, it looks like you are testing both the `merge` and `push` options in this describe block. I think this should be broken up into two describe blocks.